codebuild/cd/generic-unix-build.sh

```diff
@@ -12,24 +12,25 @@ if [ -z "$AWS_CRT_TARGET" ]; then
     AWS_CRT_TARGET=$AWS_CRT_HOST
 fi
 
-LIB_PATH=target/cmake-build/lib
 SKIP_INSTALL=
 
-# Cross compiles do not need local installs, and they have a different lib output path
 if [[ "$AWS_CRT_TARGET" != "$AWS_CRT_HOST" ]]; then
     SKIP_INSTALL=--skip-install
-    LIB_PATH=target/cmake-build/aws-crt-java/lib
```

Comment:
I did `"mv target/cmake-build/aws-crt-java/* target/cmake-build/"` in builder.json to replace this